modules/passkey/src/utils/webauthn.ts

```diff
@@ -0,0 +1,200 @@
+import CBOR from 'cbor'
+
+/**
+ * Returns the flag for the user verification requirement.
+ *
+ * See: <https://w3c.github.io/webauthn/#enumdef-userverificationrequirement>
+ */
+export function userVerificationFlag(userVerification: UserVerificationRequirement = 'preferred'): number {
+  switch (userVerification) {
+    case 'preferred':
+      return 0x01
+    case 'required':
+      return 0x04
+    default:
+      throw new Error(`user verification requirement ${userVerification} not supported`)
+  }
+}
+
+/**
+ * Encode bytes using the Base64 URL encoding.
+ *
+ * See <https://www.rfc-editor.org/rfc/rfc4648#section-5>
+ *
+ * @param data data to encode to `base64url`
+ * @returns the `base64url` encoded data as a string.
```

Comment:
I think other methods are missing the `@param` stuff.

Reply:
Updated 👍🏾